### PR TITLE
Clean out the static frontend

### PR DESCRIPTION
**Pull request by jonkafton**
### What are the relevant tickets?

https://github.com/mitodl/hq/issues/5846

### Description (What does it do?)

Removes the ./frontends/mit-learn workspace and any references. This was the entrypoint for the previous static frontend, now replaced by the Next.js entrypoint at ./frontends/main.

Also removes the E2E/browser tests as these have been inactive and have not been migrated to work with Next.js.

### How can this be tested?

Generally, all tests should pass on frontend and backend and services should build and run with Docker without error.

### CODE REVIEW

File: main/settings.py [hunk 1]
```diff
@@ -217,7 +217,6 @@
         "BACKEND": "django.template.backends.django.DjangoTemplates",
         "DIRS": [
             BASE_DIR + "/templates/",
-            BASE_DIR + "/frontends/mit-learn/build",
```
Comment (jonkafton) [inline main/settings.py:220]: Is this needed for anything?

File: scripts/test/python_tests.sh [hunk 2]
```diff
@@ -12,10 +12,6 @@ function run_test {
 	poetry run "$@"
 }
 
-# Placeholder build for use in django tests.
-mkdir ./frontends/mit-learn/build
-touch ./frontends/mit-learn/build/index.html
```
Comment (jonkafton) [inline scripts/test/python_tests.sh:17]: Are these needed for anything?

File: channels/views_test.py [hunk 3]
```diff
@@ -187,7 +187,7 @@ def test_patch_channel_image(client, channel, attribute):
         os.path.dirname(__file__),  # noqa: PTH120
         "..",
         "frontends",
-        "mit-learn",
+        "main",
         "public",
         "images",
         "blank.png",
```
Comment (jonkafton) [inline channels/views_test.py:193]: I've added a blank.png at this path, but do we actually need it for this test? It's not used in the frontend.

File: main/settings.py (hunk 1 above)
Comment (ChristopherChudzicki) [reply main/settings.py:220]: It was necessary when django served the JS

File: scripts/test/python_tests.sh (hunk 2 above)
Comment (ChristopherChudzicki) [reply scripts/test/python_tests.sh:17]: Only back when django served the JS

File: channels/views_test.py (hunk 3 above)
Comment (ChristopherChudzicki) [reply channels/views_test.py:193]: It's "supposed" to be the fallback for embedly. See frontends/ol-utilities/src/learning-resources/learning-resources.ts

That said:
- we don't need the /static version
- The version we do have set isn't working right now (https://rc.learn.mit.edu/static/images/blank.png) ... unsure why

**Review by ChristopherChudzicki — APPROVED** [review]
Approved, but could you change the url in
```
const BLANK_IMAGE = "https://rc.learn.mit.edu/static/images/blank.png"
```

to use NEXT_PUBLIC_ORIGIN

**Pull request merged**